### PR TITLE
New package: rng-tools-5

### PR DESCRIPTION
**Pull request by thypon**
#1186

### CODE REVIEW

File: srcpkgs/rng-tools/files/rngd/run [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+. /etc/conf.d/rngd
+
+a=0
+n=$(nproc)
+
+while [ $a -lt $n ]
+do
+	rngd $RNGD_OPTS
+	a=`expr $a + 1`
```
Comment (xtraeme) [inline srcpkgs/rng-tools/files/rngd/run]: `a=$((a+1))`

File: srcpkgs/rng-tools/template [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+version=5
+revision=1
+build_style=gnu-configure
+makedepends="libgcrypt-devel"
+short_desc="Random number generator related utilities"
+maintainer="Andrea Brancaleoni <miwaxe@gmail.com>"
+license="GPL-3"
+homepage="http://sourceforge.net/projects/gkernel"
+distfiles="${SOURCEFORGE_SITE}/gkernel/${pkgname}-${version}.tar.gz"
+checksum=60a102b6603bbcce2da341470cad42eeaa9564a16b4490e7867026ca11a3078e
```
Comment (xtraeme) [inline srcpkgs/rng-tools/template]: `/etc/conf.d/rng` is useless, it's empty... I would remove it completely and use `<service>/conf` instead.

Comment (thypon) [inline srcpkgs/rng-tools/template]: Could you show an example?

**Comment by xtraeme** [comment]
Sure, in `service/run`:

```
[ -r conf ] && . ./conf
... ${OPTS:=defopts} ...
```

**Pull request merged**